### ISSUE TITLE
404 instead of exception when invalid event is references

### ISSUE THREAD

**Issue by stefanbethke** [devdaydresden/devday_website#153]
**Describe the bug**
Certain URL patterns trigger an exception (DoesNotExist) when trying to determine the event the invoked URL relates to.

**To Reproduce**
1. Go to http://www.devday.de/Dev%20Day%202019/ac/1680/MLFgp0Pv5_pUVUaJa77Z90reohk%3D/
2. See error

**Expected behavior**
A normal 404 response should be rendered.

**Additional context**

/srv/devday/attendee/views.py in get_context_data
```
458:        context["event"] = Event.objects.get(slug=self.kwargs.get("event"))
```
Variable | Value
-- | --
__class__ | <class 'attendee.views.CheckInAttendeeUrlView'>
context | {'event': 'Dev Day 2019',  'id': '1680',  'verification': 'MLFgp0Pv5_pUVUaJa77Z90reohk=',  'view': <attendee.views.CheckInAttendeeUrlView object at 0x7fed4bc51160>}
kwargs | {'event': 'Dev Day 2019',  'id': '1680',  'verification': 'MLFgp0Pv5_pUVUaJa77Z90reohk='}
self | <attendee.views.CheckInAttendeeUrlView object at 0x7fed4bc51160>

**Comment by stefanbethke** [comment]
On second thought, this appears to be a different problem with the generation of the check-in URL. The slug should be 'devday.19', not 'Dev Day 2019'.

**Comment by jandd** [comment]
I assume the template or view that generates the URL passes the event itself instead of event.slug to the reverse function / url template tag.

**Comment by jandd** [comment]
Fixed in 97bd5be8
